bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/ClangdConfigurationFileManager.java

```diff
@@ -68,7 +69,19 @@ public void handleEvent(CProjectDescriptionEvent event, MacroResolver macroResol
 		setCompilationDatabasePath(event.getProject(), event.getNewCProjectDescription(), macroResolver);
 	}
 
-	@Override
+	/**
+	 * Set the <code>CompilationDatabase</code> entry in the <code>.clangd</code> file which is located in the <code>project</code> root.
+	 * The <code>.clangd</code> file will be created, if it's not existing.
+	 * The <code>CompilationDatabase</code> points to the build folder of the active build configuration
+	 * (in case <code>project</code> is a managed C/C++ project).
+	 *
+	 * In the following example clangd uses the compile_commands.json file in the Debug folder:
+	 * <pre>CompileFlags: {CompilationDatabase: Debug}</pre>
+	 *
+	 * @param project managed C/C++ project
+	 * @param newCProjectDescription new CProject description
+	 * @param macroResolver helper to resolve macros in the CWD path of the builder
+	 */
 	public void setCompilationDatabasePath(IProject project, ICProjectDescription newCProjectDescription,
```

Comment:
I think, since this is no longer part of `ClangdCProjectDescriptionListener` interface, it's enough to make this method protected.